FogCreek/Glitch-Community
# Quartz church: TextInput and TextArea components

## Links
* quartz-church.glitch.me
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/374
* Specs/doc links

## GIF/Screenshots:

## Changes:
* replaced internal textInput and textAreas with shared-components versions

## How To Test:
* Click around, try changing names of things, make sure you get errors when expected

## Things left to do before deploying:
- Also have an update to shared-components out to add focus styling (will need to bump version number to get that here)
